src/features/query-bar/components/filter/Condition.js

```diff
@@ -191,22 +191,22 @@ const Condition = ({
         }
 
         const connectorOptions = getFormattedOptions(
-            [AND, OR].map(key => ({
-                displayName: key,
-                value: key,
+            [AND, OR].map(connector => ({
+                displayName: CONNECTOR_NAMES[connector],
```

Comment:
I don't think we need CONNECTOR_NAMES today. 

What do we lose by removing this?

Reply:
I can remove, just thought I'd put it there for future reference